src/platform/nrfconnect/wifi/NrfWiFiDriver.cpp

```diff
@@ -105,7 +105,7 @@ CHIP_ERROR NrfWiFiDriver::Init(NetworkStatusChangeCallback * networkStatusChange
     {
         WiFiManager::ConnectionHandling handling{ [] { Instance().OnNetworkStatusChanged(Status::kSuccess); },
                                                   [] { Instance().OnNetworkStatusChanged(Status::kUnknownError); },
-                                                  System::Clock::Timeout{ kWiFiConnectNetworkTimeoutSeconds * 1000 } };
+                                                  System::Clock::Timeout{ kWiFiConnectNetworkTimeoutSeconds } };
```

Comment:
Why this is not needed given that `using Timeout = Milliseconds32;`?

Reply:
@markaj-nordic I pushed commit addressing this comment, as we need to move forward and merge this PR.